lib/pf/cluster.pm

```diff
@@ -644,6 +645,66 @@ sub deactivate_maintenance {
     unlink($maintenance_file);
 }
 
+=head2 encryption_password
+
+Takes the active_active.password and ensures its a 56 bytes password
+Will pad the password with zeros if its less than that
+Will strip the password to 56 bytes if its more than that
+
+=cut
+
+sub encryption_password {
+    require pf::config;
+    my $password = $pf::config::Config{active_active}{password};
+    
+    my $desired_length = 56;
+
+    # Ensure it isn't more than 56 bytes
+    $password = substr($password, 0, $desired_length);
+
+    my $missing = $desired_length - length($password) - 1;
+    for my $i (0..$missing) {
```

Comment:
Doesn't seem like what you said exists in a simple form in perl (from my vast knowledge was acquired across 2 minutes of googling this)

We could do:
```
$password .= '0' x ($desired_length - length($password) - 1)
```

Which would be on a single line and be cleaner